src/common/snippets/src/lowered/pass/assign_registers.cpp

```diff
@@ -162,10 +165,12 @@ bool AssignRegisters::run(LinearIR& linear_ir) {
         }
     }
 
-    auto linescan_assign_registers = [](const decltype(live_intervals_vec)& live_intervals,
-                                        const std::set<Reg>& reg_pool) {
-        // http://web.cs.ucla.edu/~palsberg/course/cs132/linearscan.pdf
-        std::map<LiveInterval, Reg, by_ending> active;
+    auto assign_registers = [](const decltype(live_intervals_vec)& live_intervals, const std::set<Reg>& reg_pool) {
+        OV_ITT_SCOPED_TASK(ov::pass::itt::domains::SnippetsTransform, "Snippets::AssignRegisters::LinearScanOptimized")
```

Comment:
Are there any performance results which show model compilation improvement? Could you please share them with us?